torch/_torch_docs.py

```diff
@@ -2610,7 +2610,14 @@ def merge_dicts(*dicts):
 Args:
     input (Tensor): the dividend
     other (Tensor or Number): the divisor
+    rounding_mode (str, optional): Type of rounding applied to the result.
 
+    * ``"true"`` - no rounding and integers are promoted to floating point (default)
```

Comment:
"no rounding and integers are promoted to floating point (default)" -> "default behavior. Performs no rounding and, if both attr:\`input\` and attr:\`other\` are integer types, promotes the inputs to the default scalar type."